io.typora.editor.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "io.typora.editor",
+  "command": "/app/start.sh",
+  "separate-locales": false,
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=host",
+    "--share=ipc",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "typora",
+      "modules": [
+        {
+          "name": "pandoc",
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/jgm/pandoc/releases/download/2.11.2/pandoc-2.11.2-linux-amd64.tar.gz",
+              "sha256": "25e4055db5144289dc45e7c5fb3616ea5cf75f460eba337b65474d9fbc40c0fb"
+            }
+          ],
+          "buildsystem": "simple",
+          "build-commands": [
+            "install -Dm755 bin/pandoc /app/bin/pandoc"
+          ]
+        }
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://typora.io/linux/Typora-linux-x64-0.9.96.tar.gz",
+          "sha256": "cdb060acdf7cb615180197c07bcbb463ab2a45e6985e6edae1b8249179abe197"
+        },
+        {
+          "type": "script",
+          "dest-filename": "start.sh",
+          "commands": [
+            "exec /app/typora/Typora --no-sandbox $@"
```

Comment:
You should use zypak instead of disabling the sandbox. It's available in org.electronjs.Electron2.BaseApp, see https://github.com/flathub/com.irccloud.desktop/blob/master/com.irccloud.desktop.json for an example.